src/generic3/values.yaml

```diff
@@ -65,7 +64,16 @@ app:
       memory: 256M
     limits:
       cpu: 500m
-      memory: 512Mi
+      memory: 512M
+
+#  HPA Activation
+  replica:
+    enabled: "true"
+    min: 2
```

Comment:
min should be more or equal to the replica count. Can you drop the replicaCount to 2

Reply:
Okay, done